File: app/src/main/java/com/chuongvd/app/signal/ui/SplashActivity.java
package com.chuongvd.app.signal.ui;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import com.chuongvd.app.signal.R;
import com.chuongvd.app.signal.ui.auth.login.LoginActivity;

public class SplashActivity extends AppCompatActivity {
    private static final long DELAY_TIME = 2000;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        if (savedInstanceState == null) {
            SplashFragment fragment = new SplashFragment();
            getSupportFragmentManager().beginTransaction()
                    .add(R.id.fragment_container, fragment, SplashFragment.class.getName())
                    .commit();
        }
    }
}

File: app/src/main/java/com/chuongvd/app/signal/ui/SplashActivity.kt
package com.chuongvd.app.signal.ui

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.chuongvd.app.signal.R

class SplashActivity : AppCompatActivity() {

  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_splash)
    if (savedInstanceState == null) {
      val fragment = SplashFragment()
      supportFragmentManager.beginTransaction()
          .add(R.id.fragment_container, fragment, SplashFragment::class.java.name)
          .commit()
    }
  }

  companion object {
    private val DELAY_TIME: Long = 2000
  }
}
